boundary_layer_default_plugin/config/operators/ml_engine_batch_prediction.yaml

```diff
@@ -46,7 +46,7 @@ parameters_jsonschema:
         uri:
             type: string
         max_worker_count:
-            type: string
```

Comment:
hm @SahilKhanna129 can you confirm the proper setting for this field?

Reply:
The airflow pydoc **which could certainly be in error** shows it as an int, I couldn't find anything definitive in GCP docs.

https://github.com/apache/airflow/blob/1.10.3/airflow/contrib/operators/mlengine_operator.py#L136-L138

Reply:
yeah i remember there being some examples of errors like this in the docs, which were tracked down by @SahilKhanna129 .  I just don't remember the details.  I'll see if I can get confirmation from him.

Reply:
Oh actually it was @kzvezdarov who fixed this!  #12 .  I feel inclined to assume this has not been fixed in more recent airflow versions?

Reply:
You are correct, the pydoc is still in error!  I'll revert this change.

Reply:
oh, ha, now that I actually read his comment, it's clear that the only thing broken is the documentation, as the API itself wants a string.  @dossett if you can change this back, we can get this stuff merged this afternoon!

Reply:
@kzvezdarov Do you have a reference to GCP docs for this?  I'll try to get it fixed in Airflow as well.